e2e/init/init_test.go

```diff
@@ -89,6 +89,12 @@ var _ = Describe("init flow", func() {
 			}, "30s", "1s").Should(Equal(200))
 		})
 
+		It("should return a valid service discovery namespace", func() {
```

Comment:
we should prob add this into our other tests too, but I'm thinking I can do it once we merge https://github.com/aws/amazon-ecs-cli-v2/pull/879 as we add e2e tests for a backend app.